### PR TITLE
Web development - Typography , Colors , Icons , Stock Photos & Videos     fixes #81

### LINKED ISSUE #81: Lack resources in tools section for web development
The tools section of our web development resource repository is currently in need of additional resources. Your task is to add a valuable resources that can be included in the [tools](https://github.com/jfmartinz/ResourceHub/tree/main/Web%20Development#tools) section. 

You can suggest as many resources as you like.

Categories that lack resources:

- Typography
- Colors
- Icons
- Stock Photos & Videos

### PR DESCRIPTION
**Pull request by rajvinder-kaur**
### Fixes Issue
fixes #81

### Description: 
i added resources about colors,typography,icons and  Stock Photos & Videos

### Resource URL: 
typography****
https://fonts.google.com/
https://fonts.adobe.com/
https://fontsinuse.com/
https://www.typewolf.com/
https://typehunting.com/
https://typography-daily.com/
colors***
https://flatuicolors.com/
https://realtimecolors.com
https://color.adobe.com/create/color-wheel
https://paletton.com/
https://brandcolors.net/
https://colorkuler.com/
https://www.designspiration.com/
https://www.sherwin-williams.com/content/colorsnap
icons**
https://www.flaticon.com/
https://www.iconfinder.com/
https://thenounproject.com/
https://graphicburger.com/
https://mariodelvalle.github.io/CaptainIconWeb/
stock images and videos***
https://www.shutterstock.com/
https://unsplash.com/
https://www.pexels.com/
https://pixabay.com/
https://www.videezy.com/
https://mixkit.co/

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (jfmartinz) [inline .vscode/settings.json:3]: Please remove this one. Thanks

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by jfmartinz** [comment]
Also Solve the branch conflict.  If you don't know how to solve, you can go [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

File: .vscode/settings.json (hunk 1 above)
Comment (rajvinder-kaur) [reply .vscode/settings.json]: i removed the code

**Comment by rajvinder-kaur** [comment]
i solved the conflicts please review it

**Review by jfmartinz — APPROVED** [review]
looks good! Thanks @rajvinder-kaur. ❤️

**Comment by rajvinder-kaur** [comment]
can you please put a approved label on it. please it will help my pr to get selected for hactoberfest

**Comment by jfmartinz** [comment]
Sure. Thanks @rajvinder-kaur

**Comment by jfmartinz** [comment]
I am encouraging you to contribute more on the project by adding resources. Thanks

**Pull request merged**